content/tooling/wrangler/configuration/environments.md

```diff
@@ -231,8 +231,13 @@ kv-namespaces = [
 kv-namespaces = [
     { binding = "KV", id = "bd46d6484b665e6bd134b0496ad97760" }
 ]
+vars = {
```

Comment:
I'd put an example `vars` entry in the default option above (right under the kv-namespaces from line 226-228).